packages/pangea-node-sdk/src/types.ts

```diff
@@ -970,6 +970,63 @@ export namespace Vault {
       purpose: string;
       public_key?: EncodedPublicKey;
     }
+
+    /**
+     * Parameters for an encrypt/decrypt structured request.
+     */
+    export interface EncryptStructuredRequest<O = object> {
```

Comment:
Do we want it to be an object or can we do it some kind of Dictionary? 
```
export interface Dictionary {
  [key: string]: string | boolean | number | Dictionary;
}
```

Reply:
Really the API accepts any plain object, doesn't need to be string keys and the like as long as it's JSON-serializable (which is most plain JS objects). I don't think we need to be more stringent on this.